frontend/src/common/enums/ui/icon-source.enum.ts

```diff
@@ -0,0 +1,20 @@
+import { IconName } from './icon-name.enum';
+import deleteIcon from 'assets/img/delete-icon.svg';
+import editIcon from 'assets/img/edit-icon.svg';
+import keyIcon from 'assets/img/key-icon.svg';
+import reloadIcon from 'assets/img/reload-icon.svg';
+import successIcon from 'assets/img/success-icon.svg';
+import errorIcon from 'assets/img/error-icon.svg';
+import closeIcon from 'assets/img/close-icon.svg';
+
+const IconSource = {
```

Comment:
it isn't a enum. this is a map

move it to the `common/maps/ui/`

and name as 
```suggestion
const iconNameToSrc = {
```

Reply:
read more - https://binary-studio.com/2021/08/04/js-and-everyday-data-structures/